### PR TITLE
Added Lawanu Borthakur Profile

### PR DESCRIPTION
**Pull request by lawanu-tech**
Here is my profile data. I want to contribute to this project and learn more.

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/928"></a>

### CODE REVIEW

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Your profile looks great! If you could just remove the package-lock file from the pull request then that would be great. If you need help doing that then just let us know :)

File: public/data/LawanuBorthakur.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "url": "https://www.linkedin.com/in/lawanu/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/lawanu.tech/",
+      "icon": "instagram"
+    },
+    {
+      "name": "Lawanu Tech - My Youtube Channel",
```
Comment (Panquesito7) [inline public/data/LawanuBorthakur.json:23]: I think we should capitalize `t` for the `Youtube` word. :)
```suggestion
      "name": "Lawanu Tech - My YouTube Channel",
```

Comment (krishguptadev) [reply public/data/LawanuBorthakur.json:23]: I don't think that is necessary 😅

Comment (lawanu-tech) [reply public/data/LawanuBorthakur.json]: I have remove the package-lock file from the pull request and updated the name to YouTube. Is it fine now?

**Comment by emmalearnscode** [comment]
Unfortunately deleting the package-lock file doesn’t solve the problem. The best way to solve this is to copy the contents of the package-lock file in the main repo and paste it into your own package-lock file. That way the two should be identical and git won’t add the package-lock to the commit.

**Comment by lawanu-tech** [comment]
> Unfortunately deleting the package-lock file doesn’t solve the problem. The best way to solve this is to copy the contents of the package-lock file in the main repo and paste it into your own package-lock file. That way the two should be identical and git won’t add the package-lock to the commit.

Sorry, I did not get you properly. Do you mean I have to copy the file in local machine and push?

**Comment by emmalearnscode** [comment]
> Sorry, I did not get you properly. Do you mean I have to copy the file in local machine and push?

No problem, it’s an awkward thing to explain.

take the contents of this package-lock file on the main repo: https://github.com/EddieHubCommunity/LinkFree/blob/main/package-lock.json

copy it and paste it into your package-lock file then commit and push to your fork.

**Comment by lawanu-tech** [comment]
Okay fine. After copying, I have to commit and push in the same branch right? i.e AddProfile branch.

**Comment by lawanu-tech** [comment]
I have pushed the file after copying but it seems same as before and package-json file is not removed . I am bit confused whether is it okay or not?

File: package-lock.json [hunk 2]
```diff
@@ -47648,4 +47648,4 @@
       "integrity": "sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q=="
     }
   }
-}
+}
```
Comment (krishguptadev) [inline package-lock.json:47651]: This should fix it all! :D

```suggestion
}

```

Comment (lawanu-tech) [reply package-lock.json:47651]: Krish, what do you mean? Do I have to make some changes to make this pull request close?

**Pull request closed without merging**